includes/admin/class-evf-admin-menus.php

```diff
@@ -131,8 +131,27 @@ public function builder_page_init() {
 	 * Add menu item.
 	 */
 	public function entries_menu() {
-		$entries_page = add_submenu_page( 'everest-forms', esc_html__( 'Everest Forms Entries', 'everest-forms' ), esc_html__( 'Entries', 'everest-forms' ), 'manage_everest_forms', 'evf-entries', array( $this, 'entries_page' ) );
+		$menu_title                 = esc_html__( 'Entries', 'everest-forms' );
+		$unread_entries_count       = 0;
+		$unread_entries_count_label = '';
+		$forms                      = evf_get_all_forms();
+
+		// Count unread entries from all forms.
+		foreach ( $forms as $form_id => $form_title ) {
+			$entries               = evf_search_entries(
+				array(
+					'form_id' => $form_id,
+					'status'  => 'unread',
+					'limit'   => -1,
+				)
+			);
+			$unread_entries_count += count( $entries );
```

Comment:
```
if ( is_array( $entries ) ) {
    $unread_entries_count += count( $entries );
} else {
    continue;
}
```

Reply:
@coderkoala Yes that's a very good idea. Thanks for the heads up.

Resolved with 7543d2e543e5af4c12fec80096717e921a303939.